models/_sector/dex/trades/celo/_schema.yml

```diff
@@ -15,5 +15,92 @@ models:
           combination_of_columns:
             - tx_hash
             - evt_index
-            
-  - name: dex_celo_base_trades
\ No newline at end of file
+
+  - name: curvefi_celo_base_trades
+    meta:
+      blockchain: celo
+      sector: dex
+      project: curvefi
+      contributors: tomfutago
+    config:
+      tags: ['celo', 'dex', 'trades', 'curvefi']
+    description: "Curvefi Celo base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_seed:
+          seed_file: ref('curvefi_celo_base_trades_seed')
+          filter:
```

Comment:
We may have `check_dex_seed` tests here, under `*_base_trades`.

Reply:
As per your #4774 PR? I'm bit confused on that one as I see that you have there both `check_seed` and `check_dex_seed`, but they both are calling `check_seed_macro` so ultimately both checks are the same - unless I'm missing something here? Is the goal to check both `amount_original` and `amount_raw` as 2 individual checks?

Reply:
Sorry for the misleading, you can keep the PR as is.

I was thinking to recycle seed records existed in `dex_trades_seed`. I can add back tests in the future PR.

Reply:
no worries! I'll keep an eye on #4774 anyway in case there's a better way to go about utilising check seed 👍